src/docker-in-docker/install.sh

```diff
@@ -242,58 +242,61 @@ fi
 
 echo "Finished installing docker / moby!"
 
-# Install Docker Compose if not already installed and is on a supported architecture
-if type docker-compose > /dev/null 2>&1; then
-    echo "Docker Compose v1 already installed."
-else
-    target_compose_arch="${architecture}"
-    if [ "${target_compose_arch}" = "amd64" ]; then
-        target_compose_arch="x86_64"
-    fi
-    if [ "${target_compose_arch}" != "x86_64" ]; then
-        # Use pip to get a version that runs on this architecture
-        check_packages python3-minimal python3-pip libffi-dev python3-venv
-        export PIPX_HOME=/usr/local/pipx
-        mkdir -p ${PIPX_HOME}
-        export PIPX_BIN_DIR=/usr/local/bin
-        export PYTHONUSERBASE=/tmp/pip-tmp
-        export PIP_CACHE_DIR=/tmp/pip-tmp/cache
-        pipx_bin=pipx
-        if ! type pipx > /dev/null 2>&1; then
-            pip3 install --disable-pip-version-check --no-cache-dir --user pipx
-            pipx_bin=/tmp/pip-tmp/bin/pipx
-        fi
-        ${pipx_bin} install --pip-args '--no-cache-dir --force-reinstall' docker-compose
-        rm -rf /tmp/pip-tmp
+# If 'docker-compose' command is to be included
+if [ "${DOCKER_DASH_COMPOSE_VERSION}" != "none" ]; then
```

Comment:
Can you help add a test to validate this change? thanks! 

Reference docs: https://github.com/devcontainers/cli/blob/main/docs/features/test.md

Reply:
Looks good to me, thanks.
If you could also add similar tests to `docker-outside-of-docker`, it would be great!